docs/topics/tour/kotlin-tour-intermediate-extension-functions.md

```diff
@@ -86,8 +86,10 @@ fun HttpClient.post(url: String): HttpResponse = request("POST", url, emptyMap()
 ```
 {validate="false"}
 
-These `.get()` and `.post()` functions call the `request()` function with the correct HTTP method, so you don't have to.
-They streamline your code and make it easier to understand:
+These `.get()` and `.post()` functions extend the `HttpClient` class. Because they're called on an instance of the `HttpClient` class
```

Comment:
```suggestion
These `.get()` and `.post()` functions extend the `HttpClient` class. Since they're called on an instance of the `HttpClient` class
```

This feels nitpicky, "because" is just something that goes after a premise for me.

Another thing to consider is flipping the sentence around: "They can directly use ... because they are called on an instance ...", to put the new/non-obvious information at the end.